lib/dialects/postgres/query-generator.js

```diff
@@ -893,6 +893,13 @@ module.exports = (function() {
         return 'ARRAY[' + value.map(function (v) {
           return SqlString.escape(JSON.stringify(v), false, this.options.timezone, this.dialect, field);
         }, this).join(',') + ']::JSON[]';
+      } else if (value && field && field.type instanceof DataTypes.GEOMETRY) {
+        var s = "ST_GeomFromGeoJSON('" + JSON.stringify(value) + "'";
```

Comment:
Still missing the equivalent of this code for the other dialects, correct? Which converts geojson to wkt before inserting it